cargo/lib/dependabot/cargo/file_fetcher.rb

```diff
@@ -28,9 +28,8 @@ def package_manager_version
                   end
 
         {
-          ecosystem: "cargo",
           package_managers: {
-            "channel" => channel
+            "cargo" => channel
```

Comment:
I consciously renamed this from `"channel"` to `"cargo"` because that seemed more consistent with the pattern in other ecosystems.

In the long term, it probably won't matter because the entire shape of the data is changing, but in the short term it just seemed more straightforward this way.